Bottom: ed21fa4a98215d60605119e7a3faa288bd9169cb
Top:    e7cb3c06e4eb60915c28b95cb06061fbb8ab0aa0
Author: Christopher Voltz <cjunk@voltz.ws>
Date:   2017-05-01 21:58:45 -0500

Add CHANGELOG.md

---

diff --git a/CHANGELOG.md b/CHANGELOG.md
new file mode 100644
index 0000000..d727d01
--- /dev/null
+++ b/CHANGELOG.md
@@ -0,0 +1,9 @@
+# Change Log
+
+## 2017-05-01
+- Added Rakefile to automate tasks used in creating the patchset.
+- Updated to 2017-02-09 upstream version of tablecalc.
+
+## 2016-04-29
+- Added tag, tag_row, and tag_col keywords which allow one to tag a cell in a
+  table and later get the row and column of the tagged cell.
